app/event/send-batch-success-event.js

```diff
@@ -0,0 +1,25 @@
+const { processingConfig, messagingConfig } = require('../config')
+const { EventPublisher } = require('ffc-pay-event-publisher')
+const { SOURCE } = require('../constants/source')
+const { BATCH_PROCESSED } = require('../constants/events')
+
+const sendBatchSuccessEvent = async (filename) => {
+  if (processingConfig.useV2Events) {
+    await sendV2BatchSuccessEvent(filename)
+  }
```

Comment:
Variable has been changed to `useEvents` so needs updating to reflect.

Also you can probably merge the two functions in this module together.

They used to all be split because there was V1 and V2 function.  Now there's only one type of event, some of the splits we've got across the repos are unecessary.